BusinessLogic/LootLogic.cs

```diff
@@ -281,6 +285,12 @@ public LootOverviewViewModel GetOverview(int[] display, int syndicateId)
                     .Include(l => l.Campaign)
                     .Include(l => l.Holder)
                     .Include(l => l.Item)
+                    .Where(l => itemTier == 0 || l.Item.Tier == itemTier)
+                    .Where(l => itemType == 0 ||
```

Comment:
Now that you have ItemTypeFilters enum, it's time to replace the magic numbers here as well ;)

```
                    .Where(l => itemType == (int)Item.ItemTypeFilters.All ||
                        (itemType == (int)Item.ItemTypeFilters.Armor && l.Item.Type == Item.ItemType.Armor) ||
                        (itemType == (int)Item.ItemTypeFilters.ShortRangeWeapon && l.Item.WeaponRange == Item.ItemWeaponRange.Short) ||
                        (itemType == (int)Item.ItemTypeFilters.LongRangeWeapon && l.Item.WeaponRange == Item.ItemWeaponRange.Long)
                   )
```